### DISCUSSION TITLE
Warning message when using English language

### DISCUSSION THREAD

**Discussion by ephraim71** [openai/whisper#119] [General]
I tried  ` whisper test.mp3 --model tiny.en --language English`
but I get 
 `UserWarning: tiny.en is an English-only model but receipted 'English'; using English instead.
  warnings.warn(f"{model_name} is an English-only model but receipted '{args['language']}'; using English instead.")`
It still works, but that's a weird warning message.

**Comment by jongwook** [comment]
Thanks for flagging! Should be fixed in 2d3032d

**Comment by conradg** [comment]
This is still an issue in the non-CLI python version. A solution to change the Readme has been proposed [here](https://github.com/openai/whisper/pull/248), but could be better solved at source.

## problem
this  code raises `ValueError: This model doesn't have language tokens so it can't perform lang id`
```
model = whisper.load_model("base.en")
result = model.transcribe("test.mp3")
```

But this code raises no error, despite the clashing languages
```
model = whisper.load_model("base.en")
result = model.transcribe("test.mp3", language="fr")
```

Suggestion for a fix - models know if they're multilingual through the `is_multilingual` property - could they be extended to know what language they are in the case where they're monolingual, and then infer it in the first example, and raise an error in the second?
